src/Features/Core/Portable/UseConditionalExpression/ForAssignment/AbstractUseConditionalExpressionForAssignmentCodeFixProvider.cs

```diff
@@ -168,6 +168,11 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
                 return false;
             }
 
+            if (localDeclaration.IsImplicit)
+            {
+                return false;
+            }
```

Comment:
this code is in the TryFindMatchingLocalDeclarationImmediatelyAbove method.  THe purpose of the code is to find a *real* user local-decl statement right above hte if-statement so we can merge our result into it.  It was previously returning implicit statements (which VB creates), leading to bad baddy badness.